### DISCUSSION TITLE
Analyzer Service S3 support

### DISCUSSION THREAD

**Discussion by tomikonio** [reportportal/reportportal#2332] [Q&A]
Hi,
Followup question to this issue: https://github.com/reportportal/service-auto-analyzer/issues/149.
Does the analyzer support S3 now?
If not, what should we use if we are in Kubernetes and have S3 enabled?

**Comment by tomikonio** [comment]
Any idea here please?

**Reply by HardNorth** [reply]
@tomikonio It should be, with the version 5.12.0. Just put corresponding values into "MINIO_" prefixed variables and enable TLS with "MINIO_USE_TLS: true".
